be/src/olap/base_compaction.cpp

```diff
@@ -43,9 +50,26 @@ OLAPStatus BaseCompaction::compact() {
     RETURN_NOT_OK(pick_rowsets_to_compact());
     TRACE("rowsets picked");
     TRACE_COUNTER_INCREMENT("input_rowsets_count", _input_rowsets.size());
+    _tablet->set_clone_occurred(false);
+
+    return OLAP_SUCCESS;
+}
+
+OLAPStatus BaseCompaction::execute_compact() {
+    MutexLock lock(_tablet->get_base_lock(), TRY_LOCK);
+    if (!lock.own_lock()) {
+        LOG(WARNING) << "another base compaction is running. tablet=" << _tablet->full_name();
+        return OLAP_ERR_BE_TRY_BE_LOCK_ERROR;
+    }
+    TRACE("got base compaction lock");
+
+    if (_tablet->get_clone_occurred()) {
+        _tablet->set_clone_occurred(false);
```

Comment:
Add comment to explain why setting clone occurred here.

Reply:
OK.